Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Build the manager binary
+FROM golang:1.17 as builder
```

Comment:
nitpick: I would use `golang:1.17.2` and have Dependabot upgrade the image.

Reply:
I've seen somewhere that operator-sdk (operators) doesn't supportr golang 1.17.2. That's why I used 1.17